action.yml

```diff
@@ -9,6 +9,22 @@ inputs:
       MATLAB release to set up (R2020a or later)
     required: false
     default: latest
+  products:
+    description: >-
+      Array of products to install
```

Comment:
"Array" sounds like this should be specified as a YAML array. Perhaps "Products to install, specified as a list of product names separated by spaces"?